protostar-rust/src/main.rs

```diff
@@ -3,16 +3,21 @@ use camino::Utf8PathBuf;
 use clap::Parser;
 use scarb_metadata::MetadataCommand;
 
-use rust_test_runner::pretty_printing;
 use rust_test_runner::run_test_runner;
+use rust_test_runner::{pretty_printing, RunnerConfig};
 
 #[derive(Parser, Debug)]
 struct Args {
+    /// Name used to filter tests
     test_filter: Option<String>,
```

Comment:
It is not a test name actually, it mimics the behaviour of `cargo test` https://doc.rust-lang.org/rustc/tests/index.html#filters. The difference is that we match against test name only, not a full path, because since compiler builds phantom packages from test files and those paths can get [weird](https://github.com/software-mansion/protostar/pull/2036#issuecomment-1579291040). We planned to investigate on that later on

Reply:
If a further investigation's planned, there should be an issue for it.

> The difference is that we look for a substring in a test name only, not a full path

Does it refer in any way to the `exact` parameter?

Anyhow, the name "test_filter"'s not good. Filter could be anything really. If we refer to a name, we should go with "test_name", if path "test_path", etc. It feels like "utils"

Reply:
>If a further investigation's planned, there should be an issue for it.

It will either be resolved within this PR or an issue will be created.

Reply:
Will rename it

Reply:
I've added basic stripping of paths from test names, proper handling should be done in #2093 (and only after #2092 is addressed).